src/components/elements/carousel/carousel-style.tsx

```diff
@@ -3,6 +3,14 @@ import { Flex } from 'components/containers'
 import device from 'themes/device'
 import { ReactComponent as Chevron } from 'images/svg/careers/carousel-chevron.svg'
 
+type Props = {
```

Comment:
You can change this to `StyledButtonWrapperProps` we cannot use a very general type name.